### DISCUSSION TITLE
Workflow Suggestion: 1) Hyperlink within, and outside the document 2) Drag-and-drop text from external sources

### DISCUSSION THREAD

**Discussion by LHSequalsRHS** [flxzt/rnote#1195] [Ideas]
Hey,

I am not sure if I have missed something in my search. I haven't found this in the discussions. For the first one, I have inserted the picture of what I mean. For the second one, I was thinking of something like dragging and dropping citations from Zotero text (which gives the external link).

Thanks.

![image](https://github.com/user-attachments/assets/ce762f8a-94b9-4f39-9e63-c49b4fb4f28a)

**Comment by Doublonmousse** [comment]
So you'd want to have links to different places in the document ? #294 would be a good start (so that you can navigate more easily the document,and maybe #904 or #980 after that). I'm less sure about the two other issues and about having links inside the document to other pages.

For the second part, we'd need to close #913 (url links are treated as raw text for now).
But I'm not too sure it'll magically make importing citations from Zotero work. Does it not work at all right now if you drag and drop or CTRL+C CTRL+V a citation from zotero ?
I'd have to look at the clipboard data to check what zotero exposes. If there's already text or url links it'd be feasible. But if you need to build a custom plugin for this to work it's another story all together
